src/util/http/HttpServer.h

```diff
@@ -10,6 +10,7 @@
 #include <future>
 
 #include "absl/cleanup/cleanup.h"
+#include "global/RuntimeParameters.h"
```

Comment:
This header is very expensive to include. It would be nice if the implementation could be moved into a cpp file

Reply:
Thanks for having a look. The `HttpServer.h` header is now only included in `Server.cpp` so that shouldn't be that much of a problem. The HttpServer unfortunately cannot be moved to a cpp, because the whole class is templated.

Reply:
That's fine too but I was thinking about adding a helper function in a cpp and using it to call the function indirectly